src/engraving/libmscore/textlinebase.cpp

```diff
@@ -56,8 +56,8 @@ TextLineBaseSegment::TextLineBaseSegment(Spanner* sp, Score* score, ElementFlags
 TextLineBaseSegment::TextLineBaseSegment(const TextLineBaseSegment& seg)
     : LineSegment(seg)
 {
-    _text    = new Text(*seg._text);
-    _endText = new Text(*seg._endText);
+    _text    = seg._text->clone();
+    _endText = seg._endText->clone();
```

Comment:
doesn't really change anything, but in future might if _text/_endText could point to objects derived from Text, and more obvious what's being done